rust/actyx/ax/src/cmd/settings/set.rs

```diff
@@ -74,16 +74,28 @@ fn load_yml(input: String) -> ActyxOSResult<serde_yaml::Value> {
     .ax_invalid_input()
 }
 
-pub async fn run(opts: SetOpt) -> ActyxOSResult<Output> {
-    let settings = load_yml(opts.actual_opts.input)?;
+pub(crate) fn extract_set_settings_command(
```

Comment:
Q: Why use `opts` instead of `impl SetSettingsCommand { fn extract_set_settings_command(&self) {...}}`